src/harness/renderers/gl/shaders.c

```diff
@@ -50,6 +50,7 @@ const char* fs_3d = "#version 330 core\n"
                     "\n"
                     "out uint palette_index;\n"
                     "\n"
+                    "uniform vec3 pixels_transform[2];\n"
```

Comment:
why not store this as a `mat3` here instead of 2 `vec3`s?

Reply:
A texture transformation is stored as a br_matrix23 (=6item), so I thought to be smart and avoid sending the extra (0,0,1) vector to the GPU.

Reply:
true, it might be a little less data , but then on the other hand you then have to construct a new `mat3` for every texel that the shader processes ;)

Reply:
In the shader I initialize the `mat3` with zeros, but my thought process was that the optimizer will be able to optimize out 0 multiplications.
On the other hand, sending these 8 bytes will not be a performance killer.